doc/sap_host_exporter.yaml

```diff
@@ -0,0 +1,7 @@
+# sample config
+port: "9680"
+addres: "0.0.0.0"
+log-level: "info"
+sap-control-url: ""
+sap-control-user: ""
+sap-control-pass: ""
```

Comment:
perhaps is better `sap-control-password`. I think `pass` is not really full clear

Reply:
fair enough!